### ISSUE TITLE
Phrenic nerve neuropathy

### ISSUE THREAD

**Issue by alejozilva** [obophenotype/human-phenotype-ontology#11136]
**Preferred term label: Phrenic nerve neuropathy**

**Phrenic Nerve Palsy**

**Damage to one or both left and right phrenic nerves located around the lungs. These nerves control the left and right sides of the main muscle that is used for breathing:  the diaphragm (PMID: 39205612)**

**Parent term: Motor axonal neuropathy HP:0007002**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**asilvae (ORCID: 0000-0002-6488-9334)**

**Comment by pnrobinson** [comment]
@alejozilva 
Adding new term: 
- ID: HP:6001284
- label: Phrenic neuropathy
- parents: [Abnormal peripheral nervous system physiology (HP:0032120)](https://hpo.jax.org/app/browse/term/HP:0032120)
- pmid: PMID:39205612
- def: A functional deficit of the phrenic nerve. Phrenic neuropathy (PhN) is a debilitating condition characterized by impairment of the diaphragm muscle, the primary inspiratory muscle. Diaphragm muscle ultrasound is a noninvasive tool that appears to offer both high sensitivity and specificity for diagnosis. Alternatively, electrodiagnostic studies can assess phrenic nerve health. PhN is commonly diagnosed via diaphragm muscle US or fluoroscopy (sniff test) with paradoxical or significantly decreased diaphragm muscle movement.
- comment: 
- orcid: orcid:0000-0002-6488-9334
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11136
- synonyms: Phrenic nerve neuropathy(NONE)
